DSNP-Wallet/Source/Modules/TestViewController.swift

```diff
@@ -16,17 +18,21 @@ enum TestButtons: String, CaseIterable {
 }
 
 class TestViewController: ServiceViewController, UITextFieldDelegate {
-    //MARK: UI
     private lazy var primaryTextField = getTextField(placeholder: "Primary Mnemomic")
     private lazy var secondaryTextField = getTextField(placeholder: "Secondary Mnemomic")
     
-    var extrinsicManager: ExtrinsicManager?
-    
     override func viewDidLoad() {
+        viewModel?.txHandlerDelegate = self
```

Comment:
it seems like we need the view model to not be optional in order for the tx logic to work. Is there a case where we can make it not optional?

Reply:
Going to remove this logic.